.storybook/custom/custom.scss

```diff
@@ -286,6 +286,15 @@ code.docs-code-grey {
   background-color: #f8f8f8;
 }
 
+.fddocs-button-container {
```

Comment:
Won't this work if you apply it on the `fddocs-container` without adding additional class? And prob should be `fd-docs-container`?

Reply:
`fddocs-container` was already in place so I just kept it